### ISSUE TITLE
Can't create a ticket from user form

### ISSUE THREAD

**Issue by yllen** [glpi-project/glpi#1078]
freewood from Forum GLPI-Project has sent you a message. You can reply to freewood by replying to this email.
## The message reads as follows:

Follow this steps to reproduce bug.

"Administration" -> "Users" -> Enter to the user profile -> "Created tickets" -> "New ticket for this item..."
In the new ticket form try to select Category. As a result, got error - "The action you have requested is not allowed."

Affect all roles, even superadmin.

Otherwise creating ticket thru "Assistance" -> "Create ticket" is not affected to this bug.

Can you approve this bug and should i submit bugreport somewhere?
## 

Forum GLPI-Project Mailer

**Comment by yllen** [comment]
You also have PHP notice
2016-09-28 10:51:14 [8@yllen]
  *** PHP Notice(8): Undefined index: itemtype
  Backtrace :
  front/ticket.form.php:189                          
2016-09-28 10:51:14 [8@yllen]
  *** PHP Notice(8): Undefined index: items_id
  Backtrace :
  front/ticket.form.php:189

**Comment by trasher** [comment]
Sounds like this issue has already been fixed; I cannot reproduce in on the master nor the 9.1/bugfixes branch.

Could you please confirm and close the ticket?

**Comment by kvfinn** [comment]
Good afternoon.
Exactly the same error after upgrading to 9. 1
 Tried 9. 1 / bugfixes, the error does not go away

**Comment by yllen** [comment]
Seems ok in 9.1.1

**Comment by kvfinn** [comment]
Upgrade to 9. 1. 1, the same error, try the latest version from the git, no changes

**Comment by yllen** [comment]
Test done with the latest version in dev (version master of github), no problem

**Comment by kvfinn** [comment]
This only works if the comment out in the / inc / includes. php

// Security: Check HTTP_REFERRER: need to be in GLPI.
if (! defined ( 'DO_NOT_CHECK_HTTP_REFERER')
    &&! IsCommandLine ()
    && Isset ($ _ POST) && is_array ($ _ POST) && count ($ _ POST)) {
   Toolbox :: checkValidReferer ();
}

// Security: check CSRF token
if (GLPI_USE_CSRF_CHECK
    && Isset ($ _ POST) && is_array ($ _ POST) && count ($ _ POST)
    // ALL plugins need to be CSRF compliant
    / * && Plugin :: isAllPluginsCSRFCompliant () * /) {
   // No ajax pages
   if (! preg_match ( ':'. $ CFG_GLPI [ 'root_doc']. '(/ plugins / [^ /] * |) / ajax /:', $ _SERVER [ 'REQUEST_URI'])) {
      Session :: checkCSRF ($ _ POST);
   }
}

**Comment by yllen** [comment]
I think the issue is due to a plugin.
Can you test again with your plugins deactivated?

**Comment by kvfinn** [comment]
deactivated plugins, to no avail.

**Comment by tyronewyatt** [comment]
Just upgraded to GLPI 9.1.1 and I'm also experiencing this issue. 
Unfortunately this is the method I normally use to create tickets as it gives me a chance to look at the requester past tickets to check for ongoing issues.
I also tested without plugins enabled and it still occurred .

**Comment by freewood** [comment]
Hello.
After update to 9.1.2 nothing was changed, i still can't create ticket (with category) from the user profile. Even with disabled plugins.

Even more. I did those steps to reproduce the bug on a clean version of glpi.
Installed new copy of glpi with new database;
Create ticket category;
Go into the "normal" user profile;
Switch to "Created tickets" tab;
Press "New ticket for this item...";
Choose ticket category;

Got an error "The action you have requested is not allowed."


Should i provide more info about my setup to help fix this bug? If yes, which info you needed?
